packages/docs/docs/guide/pipes/manifest.mdx

```diff
@@ -48,13 +38,14 @@ export default {
 
 ## API
 
-| Option          | Type              | Description                                                                                    |
-| --------------- | ----------------- | ---------------------------------------------------------------------------------------------- |
-| output          | `string`          | The path to the manifest file.<br />Defaults to the output folder defined in your config.      |
-| createShortcuts | `boolean`         | Whether to create the shortest possible alias for an asset.<br />Defaults to `false`.          |
-| trimExtensions  | `boolean`         | Whether to trim the extensions from the asset aliases. <br />Defaults to `false`.              |
-| includeMetaData | `boolean`         | Whether to include the tags the asset has used in the manifest file. <br />Defaults to `true`. |
-| nameStyle       | `short\|relative` | The name style for the bundle names in the manifest file.<br />Defaults to `short`.            |
+| Option          | Type                                          | Description                                                                                                           |
+| --------------- | --------------------------------------------- | --------------------------------------------------------------------------------------------------------------------- |
+| output          | `string`                                      | The path to the manifest file.<br />Defaults to the output folder defined in your config.                             |
+| createShortcuts | `boolean`                                     | Whether to create the shortest possible alias for an asset.<br />Defaults to `false`.                                 |
+| trimExtensions  | `boolean`                                     | Whether to trim the extensions from the asset aliases. <br />Defaults to `false`.                                     |
+| includeMetaData | `boolean`                                     | Whether to include the tags the asset has used in the manifest file. <br />Defaults to `true`.                        |
+| nameStyle       | `short\|relative`                             | The name style for the bundle names in the manifest file.<br />Defaults to `short`.                                   |
+| srcSortOptions  | `object`, `(assetSrcs: string[]) => string[]` | A custom sort function or object configuration to sort the source order. <br />Defaults to `{ order: 'descending' }`. |
```

Comment:
Thanks for doing the tests! A quick correction though about the docs:

**Type**
`object`, `(a: string, b: string) => number`

**Description**
If an object:<br />  • order: 'ascending' or 'descending' (default: 'descending')<br />  • collatorOptions: [Intl.Collator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator/Collator#options) options for natural or locale-aware sorting.<br />If a function:<br /> a custom comparator for advanced sorting logic.